pkg/controller/master/user/user_rbac_controller.go

```diff
@@ -0,0 +1,109 @@
+package user
+
+import (
+	"github.com/rancher/harvester/pkg/apis/harvester.cattle.io/v1alpha1"
+	apisv1alpha1 "github.com/rancher/harvester/pkg/apis/harvester.cattle.io/v1alpha1"
+	ctlapisv1alpha1 "github.com/rancher/harvester/pkg/generated/controllers/harvester.cattle.io/v1alpha1"
+	"github.com/rancher/harvester/pkg/indexeres"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	ctlrbacv1 "github.com/rancher/wrangler-api/pkg/generated/controllers/rbac/v1"
+	"github.com/sirupsen/logrus"
+	k8srbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	usernameLabelKey     = "harvester.cattle.io/userName"
```

Comment:
trying to keep the format to be consistent with `dash` instead of mixed camelCase and dash.